viewer/src/main/webapp/flamingo-components/package.json

```diff
@@ -7,7 +7,8 @@
     "build": "ng build",
     "build-prod": "npm run ng -- build --prod --output-hashing=none",
     "test": "ng test --project=core",
-    "lint": "ng lint",
+    "lint": "ng lint --project=bridge",
+    "lint-fix": "ng lint --project=bridge --fix",
```

Comment:
This should be project=bridge, but also core
bridge is the glue between angular and extjs, core contains the actual components

Reply:
bridge project also lints core files